node/src/main.rs

```diff
@@ -757,7 +757,7 @@ async fn main() {
             debug!(contention_logger, "Possible contention in tokio threadpool";
                                      "timeout_ms" => timeout.as_millis(),
                                      "code" => LogCode::TokioContention);
-            if timeout < Duration::from_secs(10) {
+            if timeout < Duration::from_secs(100) {
```

Comment:
We should leave that at 10s - that's a really long time to not make progress for a simple ping/pong test, and increasing it to 100s makes it just more likely that all nodes have frozen, making the whole system unresponsive. Let's just leave that at the old setting until we actually have some false alarms. If anything, I would actually shorten this even more.

Reply:
Alright, I've amended the commit to keep it at 10.